x-pack/plugins/snapshot_restore/server/routes/api/repositories.test.ts

```diff
@@ -253,14 +257,14 @@ describe('[Snapshot and Restore API Routes] Repositories', () => {
       path: addBasePath('repository_types'),
     };
 
-    it('returns default types if no repository plugins returned from ES', async () => {
+    it('returns module types and on-prem types if no repository plugins returned from ES', async () => {
```

Comment:
Should this description be reworded to reflect that this is only for on-prem (i.e., it's not necessarily specific to whether a plugin is installed or not)?

Reply:
Yeah, good point. I'm adding a describe block for all repo types tests (except the one with throw) to give it a `on-prem` label because responses on Cloud will all be different.